### ISSUE TITLE
Add some Interactive features in stats portion

### ISSUE THREAD

**Issue by shubh4122** [gnome-pomodoro/gnome-pomodoro#611]
## Is your proposal related to a problem?

NO

## Describe the solution you'd like

Adding a feature in the stats section which facilitates users to know exact stats values by hovering over/clicking on the graphs

**Comment by kamilprusko** [comment]
Do you mean more detailed stats like time spent per app? Or in case of one day include a history/timeline, showing at what time you had pomodoros or breaks?

**Comment by shubh4122** [comment]
No, What i mean is like, suppose I spent 4 hours and 20 min working on some day, say Monday.
So when I go to the weekly stats and hover my mouse cursor over Monday's Graph, it should show the exact time that is 4 hrs and 20 min.

[Image for reference](https://www.metabase.com/learn/images/guide-to-charts/histogram.png)
When hovered over a particular histogram pillar, it shows exact value of quantities, here in pic
